src/main/kotlin/app/revanced/patches/youtube/layout/translations/TranslationsPatch.kt

```diff
@@ -1,41 +1,97 @@
 package app.revanced.patches.youtube.layout.translations
 
 import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.stringPatchOption
+import app.revanced.patches.shared.translations.APP_LANGUAGES
 import app.revanced.patches.shared.translations.TranslationsUtils.copyXml
+import app.revanced.patches.shared.translations.TranslationsUtils.updateStringsXml
 import app.revanced.patches.youtube.utils.compatibility.Constants.COMPATIBLE_PACKAGE
 import app.revanced.patches.youtube.utils.settings.SettingsPatch
 import app.revanced.util.patch.BaseResourcePatch
+import java.io.File
 
-@Suppress("unused")
+// Array of supported RVX languages, each represented by its language code.
+val LANGUAGES = arrayOf(
+    "ar",
+    "el-rGR",
+    "es-rES",
+    "fr-rFR",
+    "hu-rHU",
+    "it-rIT",
+    "ja-rJP",
+    "ko-rKR",
+    "pl-rPL",
+    "pt-rBR",
+    "ru-rRU",
+    "tr-rTR",
+    "uk-rUA",
+    "vi-rVN",
+    "zh-rCN",
+    "zh-rTW"
+)
+
+@Suppress("DEPRECATION", "unused")
 object TranslationsPatch : BaseResourcePatch(
     name = "Translations",
-    description = "Adds Crowdin translations for YouTube.",
+    description = "Add Crowdin translations for YouTube.",
     dependencies = setOf(SettingsPatch::class),
     compatiblePackages = COMPATIBLE_PACKAGE
 ) {
+    private var CustomLanguage by stringPatchOption(
+        key = "CustomLanguage",
+        default = "",
+        title = "Custom language file",
+        description = """
+            The file path to the strings.xml file.
+            Please note that applying the strings.xml file will overwrite all existing language translations.
+            """.trimIndent()
+    )
+
+    private var SelectedLanguages by stringPatchOption(
+        key = "SelectedLanguages",
+        default = LANGUAGES.joinToString(", "),
+        title = "Selected RVX languages",
+        description = "Selected RVX languages that will be added."
```

Comment:
```suggestion
        title = "RVX languages to add",
        description = "A list of languages to be added for the RVX settings, separated by commas."
```

Reply:
@inotia00 Why was this only [commited](https://github.com/inotia00/revanced-patches/pull/55/commits/c9c3c90eff3ee82cb8a455f83bc2a1172c94414f) for YTM but not for YT?

(Same question for [this](https://github.com/inotia00/revanced-patches/pull/55#discussion_r1622994242), which had the YTM equivalent [committed](https://github.com/inotia00/revanced-patches/pull/55/commits/45efad2263cff01f5473d4be0767dcea67ac41a7))

Reply:
This suggestion has been reflected